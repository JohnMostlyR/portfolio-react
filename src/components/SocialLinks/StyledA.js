import styled from 'styled-components';

const StyledA = styled.a`
  color: #575756;
  text-decoration: none;
`;

export default StyledA;