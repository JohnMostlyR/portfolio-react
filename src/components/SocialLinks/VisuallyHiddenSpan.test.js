import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import StyledA from './StyledA';

describe('StyledA', () => {
  it('Should render and match the snapshot', () => {
    const tree = renderer.create(<StyledA/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});