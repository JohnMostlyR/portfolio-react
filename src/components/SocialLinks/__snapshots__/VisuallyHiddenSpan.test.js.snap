// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledA Should render and match the snapshot 1`] = `
.c0 {
  color: #575756;
  text-decoration: none;
}

<a
  className="c0"
/>
`;
